static/app/components/onboardingWizard/taskConfig.tsx

```diff
@@ -197,25 +197,33 @@ export function getOnboardingTasks({
       location: getOnboardingInstructionsUrl({projects, organization}),
       display: true,
       SupplementComponent: withApi(
-        ({api, task, onCompleteTask}: FirstEventWaiterProps) =>
-          !!projects?.length &&
-          task.requisiteTasks.length === 0 &&
-          !task.completionSeen ? (
+        ({api, task, onCompleteTask}: FirstEventWaiterProps) => {
+          if (hasQuickStartUpdatesFeature(organization)) {
+            if (!projects?.length || task.requisiteTasks.length > 0 || taskIsDone(task)) {
+              return null;
+            }
+            return (
+              <EventWaitingIndicator
+                text={t('Waiting for error')}
+                hasQuickStartUpdatesFeature
```

Comment:
Isn't `hasQuickStartUpdatesFeature` reference to the function?
Is that what we want to pass as a prop?

Reply:
yes this is correct https://github.com/getsentry/sentry/blob/9a3d961fb32729e4601ca8a7a83e90c7414d2a89/static/app/components/onboardingWizard/taskConfig.tsx#L513-L522

Reply:
it is a prop in the `EventWaitingIndicator` component

Reply:
But the type is a `boolean` not a function?

Reply:
yes and that is correct... maybe the name was not the best, but it is temporary. before I was using `hasQuickStartUpdatesFeature={hasQuickStartUpdatesFeature(organization)}` but I introduced an `if` and we can just mark it as `hasQuickStartUpdatesFeature` . Does it make sense?

Reply:
Sorry, totally misunderstood the code, it makes sense